test/qasm2/emit/test_qasm2_emit.py

```diff
@@ -258,9 +258,7 @@ def multiline_then():
         return q
 
     target = qasm2.emit.QASM2()
-
-    with pytest.raises(InterpreterError):
-        target.emit(multiline_then)
+    target.emit(multiline_then)
```

Comment:
Maybe you can turn off the splitting here to get the error back?

Reply:
Ah yes, I removed it before adding the kwarg. Will do!